### PR TITLE
Redesigning Students Page

### PR DESCRIPTION
**Pull request by jzbouri**

### CODE REVIEW

File: src/app/(pages)/home/_components/BlockContainer.tsx
Comment (heyitsOK) [inline src/app/(pages)/home/_components/BlockContainer.tsx:1]: please change the flip conditional css on line 27 to be a ternary operator
```${flip === true ? "bg-blueprint-50" : "bg-white"}```

**Review by heyitsOK — CHANGES_REQUESTED** [review]
please add in the design changes that hendry has made on figma as well.
thanks for all your hard work!!!

File: src/app/(pages)/students/page.tsx [hunk 1]
```diff
@@ -18,28 +18,19 @@ export default async function Students() {
...
-                </p>
-              </div>
-              <div className="flex flex-col space-y-24 text-center">
-                {teams.map((team) => (
-                  <Section team={team} key={uuidv4()} />
-                ))}
-              </div>
-            </div>
-          </section>
+    <div className="bg-blueprint-100 min-h-screen overflow-x-hidden">
```
[the remark this answers is not in the document]
Comment (jzbouri) [reply src/app/(pages)/students/page.tsx:21]: Not sure why but when I change the bg to blueprint-50, it just shows up as white. Might be an issue with running it in dev, can't find any other reason for it not working
Screenshot 2024-07-18 at 1 43 06 PM

File: src/app/(pages)/students/_components/Section.tsx [hunk 2]
```diff
@@ -9,13 +10,12 @@ export type TeamDataType = {
 
 export default function Section({team}: {team: TeamDataType}) {
     return (
-        <div className="flex flex-col space-y-6">
-            <h4 className="text-2xl md:text-4xl text-white bg-blueprint  font-semibold my-4 p-3 rounded-md">{team.teamName} Team</h4>
+        <BlockContainer title={team.teamName + " Team -"} roundedCorners={true} flip shadow={true}>
```
[the remark this answers is not in the document]
Comment (jzbouri) [reply src/app/(pages)/students/_components/Section.tsx:13]: would you like me to make those changes to BlockContainer.tsx?

**Review by heyitsOK — APPROVED** [review]
looks very good and everything is responsive.
thank you jalal!

**Pull request merged**